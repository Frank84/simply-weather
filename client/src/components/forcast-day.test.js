import React from 'react';
import ForecastDay from './forecast-day';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  shallow(<ForecastDay />);
});