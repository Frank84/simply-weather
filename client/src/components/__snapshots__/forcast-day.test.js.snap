// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<section
  style={
    Object {
      "border": "solid 2px rgba(255,255,255,0.7)",
      "borderRadius": "0.6rem",
      "fontSize": "12px",
      "fontWeight": "bold",
      "margin": "0 0.5rem",
      "padding": "0.5rem",
    }
  }
>
  <header>
    Wed
  </header>
  <img
    alt="Weather Icon"
    style={
      Object {
        "margin": "auto",
      }
    }
    width="40"
  />
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <span
      style={
        Object {
          "paddingRight": "0.5rem",
        }
      }
    >
      °
    </span>
    <span
      style={
        Object {
          "color": "#999",
        }
      }
    >
      °
    </span>
  </p>
</section>
`;
